index.js

```diff
@@ -293,6 +309,14 @@ export async function onEvent(event, { config, global }) {
 
         //}
     }
+    
+    // add top level element if there is a click, change, or submit
+    if (("event" in laudspeakerPayload)) {
+        if (laudspeakerPayload['event'] === 'click' || laudspeakerPayload['event'] === 'change' || laudspeakerPayload['event'] === 'submit') {
+          laudspeakerPayload['context']['elements'] = [getElementByOrderZero(event)];
+           //laudspeakerPayload['element_0'] = getElementByOrderZero(event)
+        }
+    }
```

Comment:
Could be simplified a fair bit but it's more of a taste / readability thing
```suggestion
    if(['click', 'change', 'submit'].includes(laudspeakerPayload["event"])) {
       laudspeakerPayload['context']['elements'] = [getElementByOrderZero(event)];
    }
```